eclipse-plugin/service/cc.codechecker.service.api/src/main/java/cc/codechecker/api/runtime/CodecheckerServerThread.java

```diff
@@ -11,7 +11,8 @@
 
 public class CodecheckerServerThread {
 
-    public final int serverPort = (new Random()).nextInt(10000) + 15000;
+    public final int serverPort = (new Random()).nextInt(10000) + 5000 + Math.min(
```

Comment:
If more than one project was loaded, it could have **rarely** happened that two threads were initiated in the same timeframe and thus the Random's initial seed was the same, so the generated port had a collision and one of the threads could not start.